app/controllers/api/v1/application_controller.rb

```diff
@@ -63,6 +64,10 @@ def current_api_user
 
       protected
 
+      def set_local_authority
+        @local_authority = LocalAuthority.find_by!(subdomain: request.subdomain)
+      end
```

Comment:
There's no need for this method and the `before_action` - the base `ApplicationController` has a `current_local_authority` method